### ISSUE TITLE
Failing tests with node 0.12.7

### ISSUE THREAD

**Issue by turboMaCk** [davewasmer/broccoli-favicon#1]
Hello!
I've forked this repo to see what is going on with https://github.com/davewasmer/ember-cli-favicon/issues/2 but Tests fails on my localhost with **node 0.12.7**. Which version are you running?

**Comment by davewasmer** [comment]
Tests fail for me too - it's been a little while since I last touched this repo. I'll have to dig in ...

**Comment by turboMaCk** [comment]
Looks like some compatibility issue with dependencies. I force versions to:

```json
  "dependencies": {
    "broccoli-caching-writer": "0.5.5",
    "chalk": "1.0.0",
    "favicons": "3.1.1",
    "hash-file": "2.0.0",
    "lodash-node": "3.9.0",
    "mkdirp": "0.5.1",
    "mocha": "2.2.5",
    "quick-temp": "0.1.2",
    "rimraf": "2.3.4",
    "rsvp": "3.0.18",
    "symlink-or-copy": "1.0.1"
  },
```
Then test one passes:
```
  broccoli-favicons
    1) creates favicons
    ✓ uses the cached favicons if persistentCacheFile option is passed in (89ms)


  1 passing (40s)
  1 failing

  1) broccoli-favicons creates favicons:
     Error: timeout of 40000ms exceeded. Ensure the done() callback is being called in this test.
```

**Comment by turboMaCk** [comment]
After some debugging – it freez on `favicons()` call in `buildFavicons` method. Act like permission issue. I also turned some logging and so. this is output from `favicons` module:

```
Setting icons...
Favicons source is an object...
Encoded to base64: test/fixtures/simple/favicon.png...
Encoded to base64: test/fixtures/simple/favicon.png...
Encoded to base64: test/fixtures/simple/favicon.png...
Encoded to base64: test/fixtures/simple/favicon.png...
Encoded to base64: test/fixtures/simple/favicon.png...
Encoded to base64: test/fixtures/simple/favicon.png...
Encoded to base64: test/fixtures/simple/favicon.png...
Encoded to base64: test/fixtures/simple/favicon.png...
Encoded to base64: test/fixtures/simple/favicon.png...

Generating favicons...
Created folder: ❤️❤️❤️❤️❤️❤️❤️❤️/tmp/core_object-cache_path-PN1CQXlx.tmp...
```

folder stat
```
device  16777220
inode   109457733
mode    16877
nlink   2
uid     501
gid     20
rdev    0
size    68
atime   1444844117
mtime   1444844117
ctime   1444844117
blksize 4096
blocks  0
link

drwxr-xr-x  2 marek  staff
```
< I do not see nothing weird here... interesting...

Then it goes to `timeOut`

**Comment by turboMaCk** [comment]
GOT IT!! Looks like my laptop is slower than you expected:D

```
    ✓ creates favicons (65361ms)
    ✓ uses the cached favicons if persistentCacheFile option is passed in (64ms)
```

Cleanup + PR

**Comment by turboMaCk** [comment]
I do not know why it passes before... I ran into much more issues lately. I really do not know if I can fix it or at least if I will have time to work on this lately :disappointed:

**Comment by davewasmer** [comment]
@turboMaCk I'm getting occasional test failures as well, with a `Uncaught Error: invalid signature: 0x6100079f`. I traced it to a dependency, and [filed an issue](https://github.com/haydenbleasel/favicons/issues/47).

The timeouts you are seeing are not surprising - I adjusted the timeout to match the latency I saw from my location, but depending on are where you are, the latency could be much higher. I bumped the timeout to 60s. Check the last section of the Readme for details on why it takes so long.
